test/intl402/NumberFormat/test-option-useGrouping.js

```diff
@@ -14,4 +15,22 @@ author: Norbert Lindenberg
 includes: [testIntl.js]
```

Comment:
The linter wants us to remove this line now :)

Reply:
OK! I'll do that :D

Reply:
Ah, looks like the error is happening in testIntl.js.

Reply:
OK, fixed. And I also added NumberFormat v3 features since this change happens with that proposal.